docker/run.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
+mkdir -p /data/cogs/Audio
+cp -u /files/Lavalink.jar /data/cogs/Audio/
```

Comment:
This shouldn't be needed, Red will fetch Lavalink on first audio load if not there.

Reply:
The problem is that the fetched jar does not support armhf out of the box and needs manual patching.
It is missing natives/linux-arm/libconnector.so. I am unsure if this can be patched in a better way or we can add the .so file to the build process as I am never touched maven in my life.

Reply:
I would suggest merging this until a arm lavalink.jar is provided as I take the one from the releases tab.